### PR TITLE
Include F(by) 2016 and Scala World 2017

### PR DESCRIPTION
**Pull request by propensive**
Includes the date for next year's Scala World (and an updated logo), and the F(by) conference in December.
Review by @SethTisue

### CODE REVIEW

File: events/_posts/2017-09-18-scalaworld.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+category: event
+title: Scala World
+logo: /resources/img/scala-world.png
+location: Lake District, UK
+description: Advanced Scala amongst beautiful scenery
```
Comment (xeno-by) [inline events/_posts/2017-09-18-scalaworld.md]: This should also mention hikes :)

Comment (propensive) [inline events/_posts/2017-09-18-scalaworld.md]: Done!

File: events/_posts/2017-09-18-scalaworld.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+category: event
+title: Scala World
+logo: /resources/img/scala-world.png
+location: Lake District, UK
+description: Advanced Scala, hiking, biking and boating in beautiful scenery
+start: 16 September 2016
```
Comment (SethTisue) [inline events/_posts/2017-09-18-scalaworld.md]: wrong year on this and the following line

**Comment by SethTisue** [comment]
fixed year

**Pull request merged**